openai/api_resources/abstract/api_resource.py

```diff
@@ -10,6 +10,7 @@
 class APIResource(OpenAIObject):
     api_prefix = ""
     azure_api_prefix = "openai"
+    azure_dalle_prefix = "dalle"
```

Comment:
If the versions are merged with the other APIs, this is probably also openai then?

Reply:
Yes. You are correct. I'll fix it.